components/BottomSheet/src/private/MDCSheetContainerView.m

```diff
@@ -328,9 +319,11 @@ - (CGFloat)maximumHeightForDraggableView:(__unused MDCDraggableView *)view {
 
 - (BOOL)draggableView:(__unused MDCDraggableView *)view
     shouldBeginDraggingWithVelocity:(CGPoint)velocity {
-  [self updateSheetState];
+  CGFloat currentSheetHeight = CGRectGetMaxY(self.bounds) - CGRectGetMinY(self.sheet.frame);
```

Comment:
Would you be willing to add a quick comment about this subtraction? It seems a bit counterintuitive to get the height by mixing the frame and bounds. I am trying to figure out how this will work as the BottomSheet "slides up" from "below" the screen.

**Is the sheet "shrunk"?**
Bounds = (0, 0, 320, 10)
Frame = (0, 470, 320, 10)
Height = 10 - 470 = -460?

**Is the sheet full-height but translated off-screen?**
Bounds = (0, 0, 320, 240)
Frame = (0, 470, 320, 240)
Height = 240 - 470 = -230?

Reply:
This math is intentional - we're calculating the height of the sheet based on how far its top edge is from the bottom of the container. Negative heights are a possibility as a result, but this won't adversely impact our logic.

Reply:
Ah yes, I missed the  *self*.bounds versus self.*sheet*.frame